Classes/Driver/AmazonS3Driver.php

```diff
@@ -307,6 +309,8 @@ public function deleteFolder($folderIdentifier, $deleteRecursively = false)
         $folderIdentifier = $this->canonicalizeAndCheckFolderIdentifier($folderIdentifier);
         $path = $this->getStreamWrapperPath($folderIdentifier);
 
+        $this->flushCacheEntriesForFolder($folderIdentifier);
```

Comment:
Op 't moment van schrijven vond ik 't prima deze items gewoon te laten verlopen, de lijst wordt niet meer gebruikt dus je hebt er geen last van. Bij nader zoeken kwam ik er helaas achter dat [garbage collection](https://stash.maxserv.com/projects/RM/repos/typo3.cms/browse/typo3/sysext/core/Classes/Cache/Backend/FileBackend.php?at=TYPO3_7-6#342) niet standaard is, dus zelf opruimen is netter, ik ga hier even verder naar kijken ;).